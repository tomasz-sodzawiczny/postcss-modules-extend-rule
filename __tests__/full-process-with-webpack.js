import fs from "fs";
import path from "path";

describe("postcss-modules-extends-rule", () => {
  it("works in full webpack process", () => {
    expect(
      fs.readFileSync(
        path.resolve(__dirname, "../__test_project__/dist/main.css"),
        {
          encoding: "utf8"
        }
      )
    ).toMatchSnapshot();
  });
});
