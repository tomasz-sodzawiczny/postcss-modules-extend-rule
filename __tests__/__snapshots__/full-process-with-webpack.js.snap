// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`postcss-modules-extends-rule works in full webpack process 1`] = `
".blueForeground_2yBawDPFBHKpX2QNNYm-Gt {
  color: blue;
}

.blueBackground_XN0vc_O5HW8pK7_q-57bt {
  background: blue;
}

.redForeground_1YmMWXl26yhA15_DBRCth1 {
  color: red;
}

.redBackground_PotKRKohNzvSAyjMcM1JQ {
  background: red;
}

.border_3zyAzxOCLrLspRdsnu9UEo {
  border: 1px solid black;
}

.blueBox_qMcEjSnNfFsFAzCgdzOSz {
  background: blue;
}

.blueBoxWithRedText_YfsRdOan0WDd7-3y4Jg3R {
  background: blue;
  color: red;
}

.redBoxWithBlueText_3nuBmEENWrAckh58_v4Qwq {
  background: red;
  color: blue;
}

.blueBoxWithBorder_3sMvZcgEgA_9-0EKz_1P7Y {
  background: blue;
  border: 1px solid black;
}

"
`;
